ion-schema/src/main/kotlin/com/amazon/ionschema/internal/util/IonValueExtensions.kt

```diff
@@ -24,9 +24,9 @@ import com.amazon.ion.IonValue
  * If this value has no annotations, returns `this`;
  * otherwise, returns a clone of `this` with the annotations removed.
  */
-internal fun IonValue.withoutTypeAnnotations() =
+internal fun <T : IonValue> T.withoutTypeAnnotations(): T =
```

Comment:
🗺️ Receiver is made generic so that when you call this function on e.g. an IonSymbol, the result is still an IonSymbol (as opposed to an IonValue that needs to be cast back to IonSymbol).